src/solvers/lowering/expr_lowering.h

```diff
@@ -12,11 +12,14 @@ Author: Michael Tautschnig
 #include <util/expr.h>
 
 class byte_extract_exprt;
+class byte_update_exprt;
 class namespacet;
 class popcount_exprt;
 
 exprt lower_byte_extract(const byte_extract_exprt &src, const namespacet &ns);
 
+exprt lower_byte_update(const byte_update_exprt &src, const namespacet &ns);
```

Comment:
Ideally should be documented. Also I don't find the commit message very clear (what's the change being made?).

Reply:
Done (all newly added functions are now documented). Rewrote the commit message as well.